src/en/infernalvoidscans/src/eu/kanade/tachiyomi/extension/en/infernalvoidscans/InfernalVoidScans.kt

```diff
@@ -1,7 +1,66 @@
 package eu.kanade.tachiyomi.extension.en.infernalvoidscans
 
 import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
 
-class InfernalVoidScans : MangaThemesia("Infernal Void Scans", "https://hivescans.com", "en") {
+class InfernalVoidScans : MangaThemesia(
+    "Infernal Void Scans",
+    "https://hivetoon.com",
+    "en",
+) {
     override val pageSelector = "div#readerarea > p > img"
+
+    override val hasProjectPage = true
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = baseUrl.toHttpUrl().newBuilder()
+            .addPathSegment(mangaUrlDirectory.substring(1))
+            .addQueryParameter("page", page.toString())
+
+        // Filters are not loaded with the ‘s’ parameter. Fix genres filter
+        if (query.isNotBlank()) {
+            url.addQueryParameter("s", query)
+        }
```

Comment:
this is the only difference, call `super.searchMangaRequest` and then remove s parameter from the request